easybuild/easyconfigs/g/gretl/gretl-2021c-foss-2020b.eb

```diff
@@ -0,0 +1,40 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Loris Bennett
+# ZEDAT, Freie Universitaet Berlin
+
+easyblock = 'ConfigureMake'
+
+name = 'gretl'
+version = '2021c'
+
+homepage = 'http://gretl.sourceforge.net'
+description = "A cross-platform software package for econometric analysis"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+# http://prdownloads.sourceforge.net/gretl/gretl-2020a.tar.xz
+source_urls = ['http://prdownloads.sourceforge.net/%(name)s']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['be468b8233b2702a15df24d1efffdf44ce082c99617cbfa01d8e2fafda370f8a']
+
+builddependencies = [
+    ('gnuplot', '5.4.1'),
+    ('libxml2', '2.9.10'),
+    ('GMP', '6.2.0'),
+    ('GtkSourceView', '3.24.11'),
+    ('ImageMagick', '7.0.10-35'),
+    ('MPFR', '4.1.0'),
+]
```

Comment:
@tardigradus I'm getting the following build error:
```
../lib/src/gretl_www.c:28:10: fatal error: curl/curl.h: No such file or directory
   28 | #include <curl/curl.h>
```

Looks like the previous version did have a dependency on `cURL`:
https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/g/gretl/gretl-2020a-foss-2019a.eb#L28
As it seems required, was there any particular reason for removing it?